sys-apps/config-site/config-site-0.ebuild

```
@@ -0,0 +1,25 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="config.site to load dropins from config.site.d"
+HOMEPAGE="https://wiki.gentoo.org/wiki/Project:Base"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+
+RDEPEND="!<sys-devel/crossdev-20230209-r1"
+
+S=${WORKDIR}
+
+src_configure() {
+	sed -e "s|@datarootdir@|${EPREFIX}/usr/share|" \
+		"${FILESDIR}/config.site.in" > config.site
+}
+
+src_install() {
+	insinto /usr/share
+	doins config.site
```

You could also do:

```suggestion
	newins - config.site < <(sed ...)
```

but up to you ;-).

---

I usually prefer to keep commands that do not require root privileges out of src_install.